src/components/templates/Jazzbar/components/JazzBarTableComponent/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./JazzBarTableComponent";
```

Comment:
Preference named exports over `default` (though I believe this file is probably irrelevant in general, RE: other comments about duplicate files)